packages/network-of-terms-catalog/catalog/datasets/abr.jsonld

```diff
@@ -41,7 +41,7 @@
         },
         {
           "@type": "FindAction",
-          "query": "file://catalog/queries/lookup/abr-cht.rq"
+          "query": "file://catalog/queries/lookup/poolparty.rq"
```

Comment:
Alternatively, the more cumbersome `abr-cht-homosaurus`, but there may be something to say against entering tech vendors in our filenames.

Reply:
Good one. I think `poolparty.rq` is fine: if we would continue with the names of the datasets, that could result in unwieldy filenames (`abr-cht-homosaurus-anothersource-yetanothersource-andyetanothersource.rq`)

Reply:
On second thought: `poolparty.rq` could suggest that this file contains one query for all sources in PoolParty. But that isn't so: e.g. the WO2 Thesaurus has its own file.

Reply:
True, but assuming query editors start looking in the `dataset-name.jsonld`, they will find the right query file there.